js/index.js

```diff
@@ -37,6 +37,73 @@ const siteContent = {
   },
 };
 
-// Example: Update the img src for the logo
-let logo = document.getElementById("logo-img");
-logo.setAttribute('src', siteContent["nav"]["img-src"])
+// helper functions 
+
+function ObjToArray(array){
+  return Object.entries(array); // returns array of key pair values
```

Comment:
```suggestion
  return Object.entries(array); // returns array of key pair values
  Object.keys(obj) //may be a method you want to check out to parse the obj keys and it returns them in an array.... it will help you separate out the keys, as I think entries returns everything...(keys and values) .. as this would make it easier since you wouldn't have to parse and filter the keys out in another step
```